src/Language/PureScript/Docs/RenderedCode/Types.hs

```diff
@@ -159,7 +161,10 @@ syntax :: String -> RenderedCode
 syntax x = RC [Syntax x]
 
 ident :: String -> RenderedCode
-ident x = RC [Ident x]
+ident x = RC [Ident x ThisModule]
+
+ident' :: String -> ContainingModule -> RenderedCode
+ident' x m = RC [Ident x m]
```

Comment:
Yes, Pursuit does use `ContainingModule` to generate links between modules in the documentation. Changes to the `purescript` library are usually no problem for Pursuit, it's just breaking changes to the JSON format that you have to be aware of (which it seems this is). But we already have a breaking change to the JSON format lined up for the next release anyway, right?